docs/MVS_Outputs.rst

```diff
@@ -155,6 +155,47 @@ To measure this, we propose to compare the energy flows in between the sectors t
 
 ** The content of this section was copied from the conference paper handed in to CIRED 2020**
 
+Self consumption (SC)
+###############################
+
+The self consumption describes the fraction of all renewable energy that is consumed by the system itself.
```

Comment:
```suggestion
The self-consumption describes the fraction of all renewable energy that is consumed by the system itself.
```

Is it really only limited to renewable energy? I would say this is about local generation in general.